src/screens/epoch/types.ts

```diff
@@ -0,0 +1,28 @@
+export type InflationType = {
+  validator: number;
+  foundation: number;
+  total: number;
+  epoch: number;
+}
+
+export type InflationGovernorType = {
+  initial: number;
+  terminal: number;
+  taper: number;
+  foundation: number;
+  foundationTerm: number;
+}
+
+export type EpochScheduleType = {
+  slotsPerEpoch: number;
+  leaderScheduleSlotOffset: number;
+  warmup: boolean;
+  firstNormalEpoch: number;
+  firstNormalSlot: number;
+}
+
+export type EpochDetailType = {
+  name: string;
+  value: number | string;
+  detail: string;
+}
```

Comment:
I would suggest putting this inside `src/screens/epoch/components/epoch_detail/types.tsx` as you're not using it anywhere else besides in that `EpochDetail`. It only goes in to `src/screens/epoch/types.ts` If someone else inside `src/screen/epoch` is using it

Reply:
Makes sense. Updated.